pgdog/src/backend/pool/inner.rs

```diff
@@ -417,6 +419,71 @@ pub(super) struct CheckInResult {
     pub(super) replenish: bool,
 }
 
+// -------------------------------------------------------------------------------------------------
+// ----- ReplicaLag --------------------------------------------------------------------------------
+
+#[derive(Clone, Copy, Debug)]
+pub enum ReplicaLag {
+    NonApplicable,
+    Duration(std::time::Duration),
+    Bytes(u64),
+    Unknown,
+}
+
+impl ReplicaLag {
+    pub fn simple_display(&self) -> String {
```

Comment:
It's cool to keep this for now but keep in mind that stats will be read by some kind of monitoring agent and it'll expect to have a consistent unit.

Since our lowest denomination is millisecond, we should probably just display it as a number and document that it's milliseconds.